From: Michael Pyne <mpyne@kde.org>
Date: Sun, 09 Aug 2015 03:17:00 +0000
Subject: Refix the kmixui_rc install path.
X-Git-Url: http://quickgit.kde.org/?p=kmix.git&a=commitdiff&h=30054aabbf94f526a3162e12bee0bcd9dc41d204
---
Refix the kmixui_rc install path.

I suppose this might break KF5 but it looks like that's still broken
anyways.
---


--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -387,9 +387,9 @@
 install( FILES restore_kmix_volumes.desktop  DESTINATION ${AUTOSTART_INSTALL_DIR})
 install( FILES kmix_autostart.desktop  DESTINATION ${AUTOSTART_INSTALL_DIR})
 if (KMIX_KF5_BUILD)
+   install( FILES kmixui.rc  DESTINATION  ${KXMLGUI_INSTALL_DIR}/kmix)
+else()
    install( FILES kmixui.rc  DESTINATION  ${DATA_INSTALL_DIR}/kmix )
-else()
-   install( FILES kmixui.rc  DESTINATION  ${KXMLGUI_INSTALL_DIR}/kmix)
 endif()
 install( FILES kmixctrl_restore.desktop  DESTINATION  ${SERVICES_INSTALL_DIR} )
 install( FILES dbus/org.kde.kmix.control.xml DESTINATION ${DBUS_INTERFACES_INSTALL_DIR} )

